huxley/www/js/components/AdvisorProfileView.js

```diff
@@ -20,6 +21,7 @@ var User = require('utils/User');
 var _handleChange = require('utils/_handleChange');
 
 require('css/Table.less');
+const AdvisorProfileViewMarkdown = require('markdown/AdvisorProfileView.md');
```

Comment:
nit: I'd place this in a folder called `text` instead of `markdown`, so we're not implying we're tied to a particular implementation. Similarly to how our LESS files are in `css`. I'd also call this variable `AdvisorProfileViewText`.